pkg/extensions/search/resolver.go

```diff
@@ -544,9 +545,120 @@ func repoListWithNewestImage(
 		return &gql_generated.PaginatedReposResult{}, err
 	}
 
+	bookmarkedRepos, err := repoDB.GetBookmarkedRepos(ctx)
```

Comment:
I would have preferred to do this operation internally of RepoDB but the current returned model doesn't allow a very clean way to do it.

Reply:
I think this specific function is good enough for now.